tests/conftest.py

```diff
@@ -25,11 +25,15 @@ def docker_env_vars(monkeypatch):
     monkeypatch.setenv('OIDC_CLIENT_ID', 'test_oidc_client_id')
     monkeypatch.setenv('OIDC_CLIENT_SECRET', 'test_oidc_secret')
 
+    # FIXME: this logic branch needs a comment to explain why we don't use the
+    #  local env.
     if in_circleci():
         import socket
         monkeypatch.setenv('DEBUG', 'True')
         monkeypatch.setenv('HOST', '0.0.0.0')
         monkeypatch.setenv('PORT', '7777')
+        # FIXME: I need this variable set for pytest+pycharm. Why not set it
+        #  for all envs?
```

Comment:
I believe this was all due to the port from circle CI 1.0 to 2.0. The way Dylan handled that meant that `docker-compose` wasn't being used to run the tests (see https://github.com/mozilla-conduit/lando-ui/blob/master/.circleci/config.yml).

I made things a little simpler in Lando API by unifying the prod and dev containers, but didn't get around to this for lando UI. For more context see https://github.com/mozilla-conduit/lando-api/commit/9e94099e5adac8fd2760246beca686ff659256a5#diff-1d37e48f9ceff6d8030570cd36286a61 and https://phabricator.services.mozilla.com/D6085